pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -183,5 +183,7 @@ def get_download_url(self, obj):
         """
         host = settings.CONTENT_ORIGIN.strip("/")
         prefix = settings.CONTENT_PATH_PREFIX.strip("/")
-        base_path = self.context["content_artifact"].relative_path.lstrip("/")
-        return f"{host}/{prefix}/{base_path}"
+        distro_base_path = self.context["path"]
+        filename_path = self.context["content_artifact"].relative_path.lstrip("/")
+        download_url = f"{host}/{prefix}/{distro_base_path}/{filename_path}"
```

Comment:
This still uses the original `CONTENT_ORIGIN`, which isn't required to have a scheme.  Should this be switched to a different config that does intentionally use a scheme?  `v2` uses `ANSIBLE_CONTENT_HOSTNAME` for building the `download_url`.

Reply:
I agree thank you for pointing this out.